packages/bundler/src/Bundler.ts

```diff
@@ -139,7 +137,10 @@ export class Bundler implements IBundler {
     const response: UserOpResponse = {
       userOpHash: sendUserOperationResponse.result,
       wait: (confirmations?: number): Promise<UserOpReceipt> => {
-        const provider = new JsonRpcProvider(RPC_PROVIDER_URLS[chainId]);
+        const providerClient = createPublicClient({
+          chain: getChain(chainId) || mainnet,
```

Comment:
Should this default to mainnet here if there's no chainId? Might be better to just error?

Reply:
Makes sense, updating